### PR TITLE
feat: og image 

### PR DESCRIPTION
**Pull request by bearsyankees**
branch of #1642, we can merge that one if its ready (or if vercel gets back to us) first, but should figure out how to approach the og image here. Bun does not support the canvas library yet but thats probs our best bet...

### CODE REVIEW

File: api/src/server.ts [hunk 1]
```diff
@@ -1,35 +1,35 @@
+import * as Sentry from '@sentry/node';
```
Comment (Josh-Cena) [inline api/src/server.ts:1]: Why are stuff moved around? Makes it hard to read the diff

Comment (bearsyankees) [reply api/src/server.ts:1]: No idea, must have been from prettier or lint or some auto formatter. Nothing purposeful

Comment (Josh-Cena) [reply api/src/server.ts:1]: Hmmm, yeah, looks like some sort of import sorting got run here.

**Comment by bearsyankees** [comment]
eh we can re do this now

**Pull request closed without merging**